integration-cli/docker_cli_rmi_test.go

```diff
@@ -152,7 +152,7 @@ func (s *DockerSuite) TestRmiImageIDForceWithRunningContainersAndMultipleTags(c
 
 	out, _, err := dockerCmdWithError("rmi", "-f", imgID)
 	// rmi -f should not delete image with running containers
-	c.Assert(err, checker.NotNil)
+	c.Assert(err, checker.IsNil)
```

Comment:
Can you cherry-pick the upstream commit to make rebasing easier?

Reply:
At the meanwhile I live my change out.